src/Microsoft.DncEng.Configuration.Extensions/LocalDevTokenCredential.cs

```diff
@@ -136,7 +134,7 @@ public override async ValueTask<AccessToken> GetTokenAsync(TokenRequestContext r
             _account = (await _app.GetAccountsAsync()).FirstOrDefault();
             if (_account == null || uiRequired)
             {
-                var deviceCodeResult = await _app.AcquireTokenWithDeviceCode(requestContext.Scopes, DeviceCodeResultCallback)
```

Comment:
Looks like we can also delete the `DeviceCodeResultCallback` implementation since it's only used here